server/workflow/workflow_server.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/argoproj/argo/pkg/apis/workflow"
 	"github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
 	"github.com/argoproj/argo/server/auth"
+	globalUtil "github.com/argoproj/argo/util"
```

Comment:
```suggestion
	argoutil "github.com/argoproj/argo/util"
```

Is a better name here